CMakeLists.txt

```diff
@@ -147,6 +147,10 @@ if(WIN32)
 	list(APPEND CORE_VFS_SRC ${CMAKE_SOURCE_DIR}/src/util/vfs/vfs-fd.c ${CMAKE_SOURCE_DIR}/src/platform/windows/vfs-w32.c)
 	file(GLOB OS_SRC ${CMAKE_SOURCE_DIR}/src/platform/windows/*.c)
 	source_group("Windows-specific code" FILES ${OS_SRC})
+	if(MSVC)
+		include_directories(SYSTEM ${CMAKE_SOURCE_DIR}/src/platform/msvc)
```

Comment:
Because this is only for MSVC. MinGW and friends have their own getopt.h, and I didn't even test with those; for all I can tell this getopt was designed for and only for MSVC.